programs/mpl-core/src/processor/write_external_plugin_adapter_data.rs

```diff
@@ -1,36 +1,36 @@
 use borsh::{BorshDeserialize, BorshSerialize};
 use solana_program::{account_info::AccountInfo, entrypoint::ProgramResult};
 
-use crate::{error::MplCoreError, plugins::ExternalPluginKey};
+use crate::{error::MplCoreError, plugins::ExternalPluginAdapterKey};
 
 #[repr(C)]
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
-pub(crate) struct WriteExternalPluginDataV1Args {
-    /// External plugin key.
-    pub key: ExternalPluginKey,
+pub(crate) struct WriteExternalPluginAdapterDataV1Args {
+    /// External external plugin adapter key.
+    pub key: ExternalPluginAdapterKey,
     /// The data to write.
     pub data: Vec<u8>,
 }
 
-pub(crate) fn write_external_plugin_data<'a>(
+pub(crate) fn write_external_plugin_adapter_data<'a>(
     _accounts: &'a [AccountInfo<'a>],
-    _args: WriteExternalPluginDataV1Args,
+    _args: WriteExternalPluginAdapterDataV1Args,
 ) -> ProgramResult {
     Err(MplCoreError::NotAvailable.into())
 }
 
 #[repr(C)]
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
-pub(crate) struct WriteCollectionExternalPluginDataV1Args {
-    /// External plugin key.
-    pub key: ExternalPluginKey,
+pub(crate) struct WriteCollectionExternalPluginAdapterDataV1Args {
+    /// External external plugin adapter key.
```

Comment:
Nit: you may want to do a search for "external external" or "externalexternal" to catch any double updates like in these doc comments.